apps/judicial-system/backend/src/app/modules/subpoena/models/subpoena.model.ts

```diff
@@ -21,6 +21,20 @@ import { Defendant } from '../../defendant/models/defendant.model'
   timestamps: true,
 })
 export class Subpoena extends Model {
+  static serviceStatusText(serviceStatus: ServiceStatus) {
+    return serviceStatus === ServiceStatus.DEFENDER
+      ? 'Birt fyrir verjanda'
+      : serviceStatus === ServiceStatus.ELECTRONICALLY
+      ? 'Birt rafrænt'
+      : serviceStatus === ServiceStatus.IN_PERSON
+      ? 'Birt persónulega'
+      : serviceStatus === ServiceStatus.FAILED
+      ? 'Árangurslaus birting'
+      : serviceStatus === ServiceStatus.EXPIRED
+      ? 'Rann út á tíma'
+      : 'Í birtingarferli' // This should never happen
```

Comment:
Should the comment be on line 34 instead?

Reply:
Technically yes according to the current logic but I'm guessing we will start using EXPIRED at some point, the comment is mostly to point out that there aren't any more options for the service status currently